README.md

```diff
@@ -15,40 +77,65 @@ npm run android
 # Open the app in the Expo Client in an iOS simulator
 npm run ios
 
-# Open the app in a web browser
-npm run web
+# Run the automated tests (prettier, eslint, and jest)
+npm test
+
+# Verify ESLint rules are being met
+npm run eslint
+
+# Rewrite files using prettier rules
+npm run prettier
 
-# Create Xcode and Android Studio projects for the app
+# Verify Prettier rules are being met
+npm run pretest
+
+# Create Xcode and Android Studio projects for the app (to be used only if Expo
+# becomes too restrictive).
 npm run eject
+```
 
-# Build a standalone IPA for your project, signed and ready for submission to
-# the Apple App Store with release-channel set to "prod".  It will be available
-# via your expo dashboard.
-yarn run build-prod-ios
+### Building for testing or production
 
-# Build a standalone APK for your project, signed and ready for submission to
-# the Google Play Store with release-channel set to "prod". It will be available
-# via your expo dashboard.
-yarn run build-prod-android
+We are using React Native so that we can build this application for both iOS
+and Android with a single data set. To create those bundles we must follow two
+similar but slightly different workflows. For both operating systems we need
+Expo to build us the appropriate bundles, an `.apk` file for Android and a
+`.ipa` file for iOS. The following commands are used for that. I've created 4
```

Comment:
**nitpick (non-blocking)**: Avoid use of first person in README's

README's typically use the "academic 'we'" or focus on statements (e.g. 'There are 4 different comments')

```suggestion
`.ipa` file for iOS. The following commands are used for that. We've created 4
```